tests/conformance/README.md

```diff
@@ -1,23 +1,58 @@
 # Conformance Tests
-// TODO update readme for conformance tests after workflow is created.
-
 ## Tests structure
 1. `tests/` directory contains the configuration and the test definition for conformance tests.
 2. All the conformance tests are within the `tests/conformance` directory.
 3. All the configurations are in the `tests/config` directory.
-4. Each of the component specific `component` definition are in their specific `component type` folder in the `tests/config` folder. For eg: `redis` statestore component definition within `state` directory. And the other component types are `state`, `secretstores`, `pubsub`. Cloud specific components will be within their own `cloud` directory within the `component type` folder eg: `pubsub/azure/servicebus`.
+4. Each of the component specific `component` definition are in their specific `component type` folder in the `tests/config` folder. For eg: `redis` statestore component definition within `state` directory. The component types are `bindings`, `state`, `secretstores`, `pubsub`. Cloud specific components will be within their own `cloud` directory within the `component type` folder eg: `pubsub/azure/servicebus`.
 5. Similar to the component definitions, each component type has its own set of the conformance tests definitions.
 6. Each `component type` contains a `tests.yml` definition that defines the component to be tested along with component specific test configuration. Nested folder names have their `/` in path replaced by `.` in the component name in `tests.yml` eg: `azure/servicebus` should be `azure.servicebus`
 7. All the tests configurations are defined in `common.go` file.
-8. Each `component type` has its own `_test` file to trigger the conformance tests.
-9. Each test added will also need to be added to the `conformance.yml` workflow file. 
+8. Each `component type` has its own `_test` file to trigger the conformance tests. Eg: `bindings_test.go`.
+9. Each test added will also need to be added to the `conformance.yml` workflow file.
+
+## Conformance test workflow
+1. All components/services needed are currently manually setup. 
+2. If the component is tested locally within a docker container requiring no secrets, then add the component to the `pr-components` step in conformance test workflow `.github/conformance.yml`. `pr-components` step generates the component matrix for which the conformance tests will be run on each PR. 
+3. If the component is tested against a service and requires secrets, then add the component to the `cron-components` step in conformance test workflow `.github/conformance.yml`. `cron-components` defines the components for which the conformance test will be run against the master code in a scehduled manner.
```

Comment:
Should this include info on how to setup a component? Points 2 and 3, to me, read as if adding it to cron-components will just make it magically work without any setup. 

If we're the ones who have to setup the component for automated testing, we can just say that too.

Reply:
this is specific to modifying the workflow. 
Also see the first point
`1. All components/services needed are currently manually setup. `

Reply:
I still think it'd be good to state that it's the dapr team's responsibility to setup the components for automated tests.

Reply:
Changed ... see now